runtime/src/mock.rs

```diff
@@ -0,0 +1,221 @@
+			.base_block(0)
+			.for_class(DispatchClass::all(), |weights| {
+				weights.base_extrinsic = EXTRINSIC_BASE_WEIGHT.with(|v| *v.borrow()).into();
+			})
+			.for_class(DispatchClass::non_mandatory(), |weights| {
+				weights.max_total = 1024.into();
+			})
+			.build_or_panic()
+	}
+}
+parameter_types! {
+	pub const BlockHashCount: u64 = 250;
+}
+impl frame_system::Config for Runtime {
+	type BaseCallFilter = frame_support::traits::Everything;
+	type BlockWeights = BlockWeights;
+	type BlockLength = ();
+	type DbWeight = ();
+	type Origin = Origin;
+	type Index = u64;
+	type BlockNumber = u64;
+	type Call = Call;
+	type Hash = H256;
+	type Hashing = BlakeTwo256;
+	type AccountId = u64;
+	type Lookup = IdentityLookup<Self::AccountId>;
+	type Header = Header;
+	type Event = Event;
+	type BlockHashCount = BlockHashCount;
+	type Version = ();
+	type PalletInfo = PalletInfo;
+	type AccountData = pallet_balances::AccountData<u64>;
+	type OnNewAccount = ();
+	type OnKilledAccount = ();
+	type SystemWeightInfo = ();
+	type SS58Prefix = ();
+	type OnSetCode = ();
+}
+parameter_types! {
+	pub const ExistentialDeposit: u64 = 1;
+}
+parameter_types! {
+	pub const UncleGenerations: u64 = 5;
+}
+
+impl pallet_authorship::Config for Runtime {
+	type FindAuthor = AuthorGiven;
+	type UncleGenerations = UncleGenerations;
+	type FilterUncle = ();
+	type EventHandler = ();
+}
+pub type ConsensusEngineId = [u8; 4];
+const TEST_ID: ConsensusEngineId = [1, 2, 3, 4];
+pub struct AuthorGiven;
+impl FindAuthor<u64> for AuthorGiven {
+	fn find_author<'a, I>(digests: I) -> Option<u64>
+	where
+		I: 'a + IntoIterator<Item = (ConsensusEngineId, &'a [u8])>,
+	{
+		for (id, data) in digests {
+			if id == TEST_ID {
+				return u64::decode(&mut &data[..]).ok()
+			}
+		}
+		None
+	}
+}
+impl pallet_balances::Config for Runtime {
+	type Balance = u64;
+	type Event = Event;
+	type DustRemoval = ();
+	type ExistentialDeposit = ExistentialDeposit;
+	type AccountStore = System;
+	type MaxLocks = ();
+	type MaxReserves = ();
+	type ReserveIdentifier = [u8; 8];
+	type WeightInfo = ();
+}
+parameter_types! {
+	pub const TreasuryPalletId: PalletId = PalletId(*b"py/trsry");
+}
+impl pallet_treasury::Config for Runtime {
+	type PalletId = TreasuryPalletId;
+	type Currency = Balances;
+	type ApproveOrigin = frame_system::EnsureSigned<u64>;
+	type RejectOrigin = frame_system::EnsureSigned<u64>;
+	type Event = Event;
+	type OnSlash = ();
+	type ProposalBond = ();
+	type ProposalBondMinimum = ();
+	type SpendPeriod = ();
+	type Burn = ();
+	type BurnDestination = ();
+	type SpendFunds = ();
+	type WeightInfo = pallet_treasury::weights::SubstrateWeight<Runtime>;
+	type MaxApprovals = ();
+}
+impl WeightToFeePolynomial for WeightToFee {
+	type Balance = u64;
+	fn polynomial() -> WeightToFeeCoefficients<Self::Balance> {
+		smallvec![WeightToFeeCoefficient {
+			degree: 1,
+			coeff_frac: Perbill::zero(),
+			coeff_integer: WEIGHT_TO_FEE.with(|v| *v.borrow()),
+			negative: false,
+		}]
+	}
+}
```

Comment:
Another issue is are we going to use this mocked one or a real one here

Reply:
From other projects' usage, they simply do not have an unit test for runtime. I will suggest a full real mock test will target everything. 

In our meeting last time, we only want to have a transaction fee related test.